crates/lsp/src/providers/completion.rs

```diff
@@ -122,18 +132,26 @@ pub(crate) fn completion(
                         } else {
                             // if parent_parent_node.is_some() && parent_parent_node.unwrap().kind() ==
                             // "posting_or_kv_list" {
-                            complete_account(snapshot.beancount_data)
+                            complete_account_with_prefix(snapshot.beancount_data, &prefix)
                             //} else {
                             //    Ok(None)
                         }
                     }
-                    "narration" => {
-                        debug!("providers::completion - handle node - handle narration");
-                        complete_narration(snapshot.beancount_data)
-                    }
                     "payee" => {
                         debug!("providers::completion - handle node - handle payee");
-                        complete_narration(snapshot.beancount_data)
+                        complete_narration_with_quotes(
```

Comment:
I haven't changed this from the cheery-picked commit, but it is matching for `payee` and competing `narration`?